google-cloud-bigtable/Rakefile

```diff
@@ -174,6 +174,34 @@ task :doctest do
   sh "bundle exec yard config load_plugins true && bundle exec yard doctest"
 end
 
+desc "Run rubocop on yard examples."
+task :rubocop_yard_examples do
+  YARD.parse "lib/**/*.rb"
+  registry = YARD::Registry.load_all
+  examples = registry.all.map { |object| object.tags(:example) }.flatten
+
+  dir_name = "rubocop_yard_examples"
+  rm_rf dir_name if File.exist? dir_name
+  mkdir_p dir_name
+  hsh = examples.each_with_object({}) do |e, h|
+    obj = e.object
+    # puts "\n\n"
+    # puts "location: #{obj.file}:#{obj.line}"
+    # puts "example: #{e.name}" if e.name&.length > 0
+    # puts "\n\n"
+    name = "#{obj.file.gsub %r{(\.|/|-)}, '_'}_#{obj.line}"
+    h[name] ||= []
+    h[name] << e.text
```

Comment:
Under what conditions could there be multiple example tags at the same file and line?

Reply:
It happens whenever class or method docs have more than one `@example` tag. The line number points to the first line of the declaration of the code object, not to the start of the example, unfortunately.